tools/wheel_setup.py

```diff
@@ -0,0 +1,67 @@
+import os
+from setuptools import setup
+from setuptools.dist import Distribution
+import sys
+
+NAME = 'pygmo'
+VERSION = '@pygmo_VERSION@'
+DESCRIPTION = 'Parallel optimisation for C++ / Python'
+LONG_DESCRIPTION = 'A platform to perform parallel computations of optimisation tasks (global and local) via the asynchronous generalized island model.'
+URL = 'https://github.com/esa/pagmo2'
+AUTHOR = 'The pagmo development team'
+AUTHOR_EMAIL = 'pagmo@googlegroups.com'
+LICENSE = 'GPLv3+/LGPL3+'
```

Comment:
The license for pygmo has changed, now it is ``'MPL-2.0'`` (i.e., Mozilla Public License 2.0).